diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/PackageNaming.kt

```diff
@@ -60,10 +60,10 @@ class PackageNaming(private val configRules: List<RulesConfig>) : Rule("package-
         isFixMode = autoCorrect
         emitWarn = emit
 
-        val domainNameConfiguration = configRules.getRuleConfig(PACKAGE_NAME_MISSING)?.configuration
+        val domainNameConfiguration = configRules.getCommonConfig()?.configuration
         if (domainNameConfiguration == null) {
-            log.error("Not able to find an external configuration for domain name in the configuration of" +
-                " ${PACKAGE_NAME_MISSING.name} check (is it missing in json config?)")
+            log.error("Not able to find an external configuration for domain name in the common configuration " +
+                    "check (is it missing in yml config?)")
```

Comment:
```suggestion
            log.error("Not able to find an external configuration for domain name in the common configuration (is it missing in yml config?)")
```
'check' corresponded to `PACKAGE_NAME_MISSING` too